test/payment/payment-intent-generator.spec.js

```diff
@@ -0,0 +1,42 @@
+import paymentRequestDataMock from '../mocks/payment-request-data';
+import PaymentIntentGenerator from '../../src/payment/payment-intent-generator';
+
+describe('PaymentIntentGenerator', () => {
+    let data;
+    let urlHelper;
+    let requestSender;
+    let paymentIntentMapper;
+    let paymentIntentGenerator;
+
+    beforeEach(() => {
+        data = paymentRequestDataMock;
+
+        urlHelper = {
+            getGeneratePaymentIntentUrl:
+                jasmine.createSpy('getPaymentUrl').and.returnValue('}/api/v2/public/payments/payment_intents'),
```

Comment:
There is a `}` that I guess is not part of the url path
```suggestion
                jasmine.createSpy('getPaymentUrl').and.returnValue('/api/v2/public/payments/payment_intents'),
```

Reply:
done